apps/api/e2e/utils/api-types.ts

```diff
@@ -658,3 +658,32 @@ export interface CompetitionLeaveResponse extends ApiResponse {
   success: true;
   message: string;
 }
+
+// Agent rank (global rankings)
+export interface AgentRank {
```

Comment:
some type to encapsulate the `AgentRank` that will eventually come from a new `leaderboards` or `rankings` table + elo score (tbd on future schema).